src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/GithubAuthenticator.kt

```diff
@@ -1,28 +1,22 @@
 package org.vaccineimpact.orderlyweb.security.authentication
 
-import org.eclipse.egit.github.core.User
-import org.eclipse.egit.github.core.client.GitHubClient
-import org.eclipse.egit.github.core.client.RequestException
-import org.eclipse.egit.github.core.service.OrganizationService
-import org.eclipse.egit.github.core.service.TeamService
-import org.eclipse.egit.github.core.service.UserService
 import org.pac4j.core.context.WebContext
 import org.pac4j.core.credentials.TokenCredentials
 import org.pac4j.core.credentials.authenticator.Authenticator
 import org.pac4j.core.exception.CredentialsException
 import org.pac4j.core.profile.CommonProfile
-import org.pac4j.core.util.CommonHelper
-import org.slf4j.LoggerFactory
 import org.vaccineimpact.orderlyweb.db.AppConfig
 import org.vaccineimpact.orderlyweb.db.Config
 import org.vaccineimpact.orderlyweb.db.UserRepository
-import org.vaccineimpact.orderlyweb.errors.BadConfigurationError
 import org.vaccineimpact.orderlyweb.models.UserSource
+import org.vaccineimpact.orderlyweb.security.providers.GithubApiClientAuthHelper
+import org.vaccineimpact.orderlyweb.security.providers.GithubAuthHelper
 
 
 class GithubAuthenticator(private val userRepository: UserRepository,
```

Comment:
I've pulled out the code that was checking for team membership etc from this class and into the helper class which is also used by the new GithubOAuthAuthenticator and GithubOAuthProfileCreator classes